packages/react-native/ReactCommon/react/renderer/textlayoutmanager/platform/android/react/renderer/textlayoutmanager/TextLayoutManager.cpp

```diff
@@ -201,6 +202,36 @@ LinesMeasurements TextLayoutManager::measureLines(
   return lineMeasurements;
 }
 
+LinesMeasurements TextLayoutManager::measureLines(
+    const AttributedString& attributedString,
+    const ParagraphAttributes& paragraphAttributes,
+    Size size) const {
+  auto lineMeasurements = lineMeasureCache_.get(
+      {attributedString, paragraphAttributes, size},
+      [&](const LineMeasureCacheKey& /*key*/) {
+        auto measurement =
+            doMeasureLines(attributedString, paragraphAttributes, size);
+
+        return measurement;
+      });
+
+  return lineMeasurements;
+}
+
+float TextLayoutManager::getLastBaseline(
```

Comment:
I think this name is no longer up to date with current impl which returns first baseline on Android and iOS.

Reply:
+1 also just calling this `baseline` as the other methods feels better 👍

Reply:
Yes, I must have forgotten to rename it, done in https://github.com/facebook/react-native/pull/45102/commits/3fca45065dd60158d0f138bd283c85b1d4e50131